### PR TITLE
Update SUMMARY.md

### PR DESCRIPTION
**Pull request by nahern**
request to re-order sections in this doc, as requested by Bruno Woltzenlogel-Paleo

### CODE REVIEW

File: doc/SUMMARY.md [hunk 1]
```diff
@@ -2,31 +2,31 @@
...
-
 - [jcli](./jcli/introduction.md)
     - [Cryptographic keys](./jcli/key.md)
     - [Address](./jcli/address.md)
     - [Transaction](./jcli/transaction.md)
     - [Certificate](./jcli/certificate.md)
     - [Genesis](./jcli/genesis.md)
     - [REST](./jcli/rest.md)
+    
+    - [staking and stake pool](./stake_pool/introduction.md)
```
Comment (NicolasDP) [inline doc/SUMMARY.md]: this cannot be part of the JCLI chapter.

have you tried generating the output of this? It is pretty straightforward: [install mdbook](https://github.com/rust-lang-nursery/mdBook#installation)

Comment (nahern) [reply doc/SUMMARY.md]: can we move the staking chapter below the JCLI chapter please

Comment (ceilican) [reply doc/SUMMARY.md]: I think the indentation spaces here are just a typo. 

@nahern , could you remove the indentation spaces from this line?

Apart from that, I think this PR is ready to be accepted. But the Jormungandr's team should have the final decision.

Comment (nahern) [reply doc/SUMMARY.md]: this PR is to re-order sections in this doc, to move the JCLI chapter ahead of the stake pool chapter, indentation is a mistake, and I can't seem to edit this.

**Comment by vincenthz** [comment]
reordering suggestion applied with 3ba9e1c7b55c871e26e126d9c41aab20cbbd2f46

**Pull request closed without merging**